packages/deployer/internal/contract-helper.js

```diff
@@ -69,10 +70,33 @@ function contractIsModule(contractSourcePath) {
   return source.startsWith(`${hre.config.deployer.paths.modules}${path.sep}`);
 }
 
+/**
+ * Get contract source path absolute location. Will try to look it up on the local
+ * hardat folders, or on node_modules.
+ * @param {string} contractSourcePath contract path to file, e.g.: contracts/modules/SomeModule.sol
+ * @returns {string}
+ */
+function getContractFilePath(contractSourcePath) {
```

Comment:
This function was added by @mjlescano so that the deployer can see contracts defined in dependencies, in this case `core-contracts`.